packages/x-components/src/x-modules/related-tags/components/related-tag.vue

```diff
@@ -49,24 +50,33 @@
     @State('relatedTags', 'selectedRelatedTags')
     public selectedRelatedTags!: RelatedTagModel[];
 
+    /**
+     * Generates the {@link WireMetadata | event metadata} object omitting the moduleName.
+     *
+     * @returns The {@link WireMetadata} object omitting the moduleName.
+     * @internal
+     */
+    protected eventMetadata(): Omit<WireMetadata, 'moduleName'> {
```

Comment:
Rename to `createEventMetadata`. Methods should start with a verb if we follow our `base-naming` guide.